Problem 2.java

```diff
@@ -0,0 +1,32 @@
+// Time Complexity : O(n)
+// Space Complexity : O(1)
```

Comment:
recheck SC

Reply:
I fixed is now, its O(h) where h is the height of the tree